static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -40,6 +40,15 @@ function renderString(arg: string | number | boolean | Object) {
 export function MessageFormatter({breadcrumb}: MessageFormatterProps) {
   let logMessage = '';
 
+  if (!breadcrumb.data?.arguments) {
+    // There is a possibility that we don't have arguments as we could be receiving an exception type breadcrumb.
+    // In these cases we just need the message prop.
+
+    // There are cases in which our prop message is an array, we want to force it to become a string
+    logMessage = breadcrumb.message?.toString() || '';
```

Comment:
NOTE: 
Exceptions breadcrumb types are returning Arrays in their message prop, which probably is an issue with the core SDK, to handle that I'm forcing it to become a string.

Reply:
We will see the same output if the input is `{message: true}` or `{message: "true"}`, both will show `true` in the output.

Similarly, if message is a plain object we're going to see `[object Object]` in the output.

But we expect message to be a string, at least that's what the TS types say. And we know the value is coming from a JSON payload, so there's not going to be `Date` objects or custom class instances in there...

based on all the this seems fine to me.